### DISCUSSION TITLE
Include a user-controlled dashboard for new installs

### DISCUSSION THREAD

**Discussion by jcushman** [home-assistant/frontend#21294] [Dashboard]
I just set up HA via a Home Assistant Green for the first time, and thought I would report on what was confusing with fresh eyes.

The good news is, most things worked well!

The one thing I would recommend to make things less confusing for new users is to **add a user-controlled dashboard to the default configuration.**

After getting my devices set up, the first thing I wanted to do was to show the sensors I cared about on a graph. How do I do that as a new user? I now know the real answer, but here's what I tried:

- There's no button to add widgets to the default dashboard.
- It's not in developer tools.
- If you go to Settings > Devices and click on a device, there's an "add to dashboard" button, but it shows the confusing message "no config found" for the only existing dashboard.
- If you got to Settings > Dashboards > Add Dashboard and add a Default Dashboard, which sounds like a good starting place, you still can't edit anything because it's not user controlled.

Of course what finally works is Settings > Dashboards > Add Dashboard > Blank dashboard, or finding the sub-menu to take control of an existing dashboard. But a person can spend a lot of time poking around and googling this before finding the solution. Having a "Custom" dashboard set up by default would have made the initial experience friendlier.

Thanks for considering!

**Comment by Wendelstein7** [comment]
Good idea! Especially the no config found error could be very confusing. As people who've used HASS for a long while you can easily oversee or overlook these newcomers issues :)

**Comment by garretthope98** [comment]
Newcomer here. This was definitely confusing for me... wasn't really able to find anything online either, other than this thread, to help me figure out why I couldn't add certain devices to my default dashboard. I think at least providing a more descriptive error message would be super helpful!
